.github/workflows/acceptance-tests.yaml

```diff
@@ -35,6 +36,7 @@ jobs:
           TF_LOG: DEBUG
           TF_ACC: 1
           TF_UPDATE_CASSETTES: false
+          TF_TEST_DOMAIN: scaleway-terraform.com
```

Comment:
Could you add some information about who is going to manage this domain name, how is it use to run the test suite?

Reply:
The domain was added to the test organisation.